### PR TITLE
removed settings.ini from repo

### PR DESCRIPTION
**Pull request by NicholasBatesNZ**
removed settings.ini from repo so that it doesn't bother you when committing changes

### CODE REVIEW

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by vampcat** [comment]
Can I get some solid answers on the comments, so I can merge this?

File: .gitignore [hunk 1]
```diff
@@ -70,4 +70,5 @@ engine/prevShip.ini
 engine/src/main/resources/prevShip.ini
 engine/settings.ini
 engine/src/main/resources/settings.ini
+engine/src/main/resources/org/destinationsol/ui/settings.ini
```
[the remark this answers is not in the document]
Comment (NicholasBatesNZ) [reply .gitignore]: I have found that when the removed settings.ini (engine/src/main/resources/settings.ini) file is not present, the game generates a new settings.ini in this location

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Pull request merged**